components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/role/edit-users.jsp

```diff
@@ -144,7 +144,7 @@
                 FlaggedName[] data = client.getUsersOfRole(roleName, filter, -1);
                 if (CarbonUIUtil.isContextRegistered(config, "/usermgt-workflow/")) {
                     String[] DeletePendingRolesList = UserMgtClient.
-                            listAllEntityNames("DELETE_USER", "PENDING", "USER");
+                            listAllEntityNames("DELETE_USER", "PENDING", "USER", UserAdminUIConstants.WILD_CARD);
```

Comment:
Shouldn't we send the filter here? It would reduce the cost of the select query.